src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -703,6 +704,23 @@ def load_command_table(self, _):
             is_preview=True)
         g.wait_command('wait')
 
+    with self.command_group('sql midb log-replay',
+                            managed_databases_operations,
+                            client_factory=get_sql_managed_databases_operations) as g:
+        g.custom_command('start', 'managed_db_log_replay_start', supports_no_wait=True)
+        g.command('stop', 'delete', confirmation=True, supports_no_wait=True)
+        g.command('complete', 'complete_restore')
```

Comment:
please add wait command here

Reply:
I added, but can you tell what is the use case for wait command? Why am I adding it (there must be some rule that I am not aware of ) ?

Reply:
because you support no wait for some commands, e.g. start, stop, if users use --no-wait in your command, the command will not wait any more. But if i want to query command status, i have to use wait command

Reply:
Oh okay I get it now. Thanks